internal/executor/util/kubernetes_object.go

```diff
@@ -66,8 +66,17 @@ func CreateIngress(name string, job *api.Job, pod *v1.Pod, service *v1.Service,
 	})
 
 	rules := make([]networking.IngressRule, 0, len(service.Spec.Ports))
-	tlsHosts := make([]string, 0, len(service.Spec.Ports))
+	tlsHosts := make([]string, 0, len(service.Spec.Ports)+1)
 
+	// First host used for certificate signing, needs to be less than 64 chars long
+	domain := executorIngressConfig.CertDomain
+	if domain == "" {
+		domain = "svc"
```

Comment:
I'm not sure if this is the right way to set the default, but I've been looking around the codebase and haven't stumbled across how it's done for HostnameSuffix

Reply:
https://github.com/G-Research/armada/blob/master/config/executor/config.yaml#L24